src/meta/processors/job/StorageJobExecutor.cpp

```diff
@@ -81,6 +81,12 @@ ErrOrHosts StorageJobExecutor::getLeaderHost(GraphSpaceID space) {
       it->second.emplace_back(partId);
     }
   }
+  // If storage has not report leader distribution to meta and we don't report error here,
+  // JobMananger will think of the job consists of 0 task, and the task will not send to any
+  // storage. And the job will always be RUNNING.
```

Comment:
Here I got an issue for @liangliangc . That Explorer will  execute "SUBMIT JOB STATS" after create space and wait a heartbeat, and job is always running, what do you think?

Reply:
BTW, I am not sure whether we should check leader count same as partition count here... WDYT?

Reply:
Maybe a flag to identify a space is ready to supply survice.
Maybe use leader count == partition,both okay for me.